presentation/frontend/src/presentation-frontend/selection/HiliteSetProvider.ts

```diff
@@ -6,7 +6,6 @@
  * @module UnifiedSelection
  */
 
-import { once } from "lodash";

```

Comment:
Perhaps we can remove dependency on `lodash` from `package.json`?

Reply:
We probably could, but I don't see that as a high priority thing to do at the moment - it's being used by some of our other dependencies as well.

Reply:
BTW, while looking at where we use `lodash` I see `PresentationPropertyDataProvider` also has a similar problem... going to fix it in this PR as well.

Reply:
And now I also see that `presentation-frontend` doesn't use `lodash` at all at the moment (was looking at `presentation-components`).. going to make the change as well.

Reply:
Fixed the problem in `PresentationPropertyDataProvider` and dropped `lodash` dependency from all presentation packages. Please review.

Reply:
Thanks